include/tcpdf_fonts.php

```diff
@@ -1995,7 +1997,7 @@ public static function getUniord($uch) {
 	 * @author Nicola Asuni
 	 * @public static
 	 */
-	public static function UTF8StringToArray($str, $isunicode=true, &$currentfont) {
+	public static function UTF8StringToArray($str, $isunicode, &$currentfont) {
```

Comment:
Is array below must check before if the value is not null to not throw a checking offset of null warning